### PR TITLE
Backend for chat

### PR DESCRIPTION
**Pull request by lerabo**

### CODE REVIEW

File: src/modules/chat-room/chat-room.service.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { ChatRoomDto } from './dto/chat-room.dto';
+
+@Injectable()
+export class ChatRoomService {
+  constructor(
+    @InjectRepository(ChatRoom)
+    private chatRoomRepository: Repository<ChatRoom>,
+  ) {}
+  async createRoom(data: ChatRoomDto): Promise<ChatRoom> {
+    return await this.chatRoomRepository.save({
```
Comment (lyudad) [inline src/modules/chat-room/chat-room.service.ts]: where try/catch

File: src/modules/chat-room/chat-room.service.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      jobPostId: { id: data.jobPostId },
+      senderId: { id: data.senderId },
+      receiverId: { id: data.receiverId },
+      activeRoom: data.activeRoom,
+    });
+  }
+
+  async updateRoom(id: number, data: Partial<ChatRoomDto>): Promise<ChatRoom> {
+    const room = await this.chatRoomRepository.findOneBy({ id });
```
Comment (lyudad) [inline src/modules/chat-room/chat-room.service.ts]: try/catch?

File: src/modules/message/message.service.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { Message } from 'src/entities/message.entity';
+
+@Injectable()
+export class MessageService {
+  constructor(
+    @InjectRepository(Message)
+    private messageRepository: Repository<Message>,
+  ) {}
+
+  async createMessage(data: MessageDto): Promise<Message> {
```
Comment (lyudad) [inline src/modules/message/message.service.ts:14]: try/catch everywhere

File: src/modules/chat-room/chat-room.service.ts (hunk 1 above)
Comment (lerabo) [reply src/modules/chat-room/chat-room.service.ts]: added

File: src/modules/chat-room/chat-room.service.ts (hunk 2 above)
Comment (lerabo) [reply src/modules/chat-room/chat-room.service.ts]: fixed

File: src/modules/message/message.service.ts (hunk 3 above)
Comment (lerabo) [reply src/modules/message/message.service.ts:14]: added

**Pull request merged**